extmod/modubinascii.c

```diff
@@ -89,14 +88,89 @@ mp_obj_t mod_binascii_unhexlify(mp_obj_t data) {
 }
 MP_DEFINE_CONST_FUN_OBJ_1(mod_binascii_unhexlify_obj, mod_binascii_unhexlify);
 
+static const char *base64codes = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789+/=";
```

Comment:
There's a reason everyone uses the static array: the base64 text codes aren't contiguous on the ASCII table, so a conversion without one means every simple lookup changes to at least one "if" statement, possibly 2 or 3. This means more code, and more branching to slow things down.

I'll give it a shot and check the results.